pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -1041,24 +1063,24 @@ created with @racket[thread/suspend-to-kill] can have zero
 custodians. Extra custodians become associated with a thread through
 @racket[thread-resume] (see @secref["threadkill"]). When a thread
 has multiple custodians, it is not necessarily killed by a
-@racket[custodian-shutdown-all], but shut-down custodians are removed
-from the thread's managing set, and the thread is killed when its
+@racket[custodian-shutdown-all]. Instead, shut-down custodians are removed
+from the thread's managing custodian set, and the thread is killed when its
 managing set becomes empty.
 
 The values managed by a custodian are only weakly held by the
 custodian. As a result, a @techlink{will} can be executed for a value that
 is managed by a custodian. In addition, a custodian only weakly
 references its subordinate custodians; if a subordinate custodian is
 unreferenced but has its own subordinates, then the custodian may be
-collected, at which point its subordinates become immediately
-subordinate to the collected custodian's superordinate custodian.
+garbage collected, at which point its subordinates become immediately
+subordinate to the collected custodian's superordinate (owner) custodian.
 
 In addition to the other entities managed by a custodian, a
 @deftech{custodian box} created with @racket[make-custodian-box]
 strongly holds onto a value placed in the box until the box's
-custodian is shut down. The custodian only weakly retains the box
-itself, however (so the box and its content can be collected if there
-are no other references to them).
+custodian is shut down. However, the custodian only weakly retains the box
```

Comment:
+1 for keeping "however" mid-sentence

Reply:
Again, what?

Reply:
I prefer the old sentence: 
> The custodian only weakly retains the box
itself, however ....

Reply:
Interesting. This is a standard copy edit so as not to bury the contrast word.